AirCasting/SDSync/ViewsFlow/SDSyncingAB/SyncingABView.swift

```diff
@@ -32,16 +32,7 @@ struct SyncingABView<VM: SDSyncViewModel>: View {
         }
         .padding()
         .background(navigationLink)
-        .onReceive(viewModel.isSyncCompleted, perform: { isConnected in
-            viewModel.presentNextScreen = isConnected
-        })
-        .onReceive(viewModel.shouldDismiss, perform: { dismiss in
-            viewModel.presentAlert = dismiss
-        })
-        .onReceive(viewModel.progress, perform: { newProgress in
-            self.progressLabel = newProgress?.progressLabel()
```

Comment:
@jankrzempek  This was needed for showing progress. Please watch out for deleting too much